pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -964,6 +967,109 @@ type MSTeamsConfig struct {
 	HTTPConfig *HTTPConfig `json:"httpConfig,omitempty"`
 }
 
+// RocketChatConfig configures notifications via RocketChat.
+// See https://prometheus.io/docs/alerting/latest/configuration/#rocketchat_config
+type RocketChatConfig struct {
+	// Whether to notify about resolved alerts.
+	// +optional
+	SendResolved *bool `json:"sendResolved,omitempty" yaml:"send_resolved,omitempty"`
+	// The API URL for RocketChat.
+	// Defaults to global.rocketchat_api_url if not specified.
```

Comment:
Please replace with the default https://open.rocket.chat/

Reply:
Hi @slashexx maybe I didn't make it clear  - "global.rocketchat_api_url" this is a direct reference from prometheus documentation, still is present. Would work better if you reference that https://open.rocket.chat is the default

Reply:
Also, this was added to the chanel instead the api, if you could fix it.
```go
// Defaults to https://open.rocket.chat/ if not specified.
```